src/components/shared/UnitCircleExercise.tsx

```diff
@@ -0,0 +1,92 @@
+import UnitCircleBase from '../../assets/unit-circle-base.svg';
+import UnitCircleMarkers from '../../assets/unit-circle-markers.svg';
+import UnitCircleTurle from '../../assets/unit-circle-turtle.svg';
+
+import '../../styles/ExerciseSide.scss';
+
+/* Currently working under the assumption that the exercies
+   has 5 total ticks. */
+
+export interface UnitCircleTick {
+  label: string;
+  type: string;
+}
+
+interface UnitCircleExerciseProps {
+  turtleAngle: number;
+  labelOne: UnitCircleTick;
+  labelTwo: UnitCircleTick;
+  labelThree: UnitCircleTick;
+  labelFour: UnitCircleTick;
+  labelFive: UnitCircleTick;
+}
+
+export function UnitCircleExercise({
+  turtleAngle,
+  labelOne,
+  labelTwo,
+  labelThree,
+  labelFour,
+  labelFive,
+}: UnitCircleExerciseProps): JSX.Element {
+  return (
+    <section id="exercise-side-container">
+      <img
+        className="unit-circle"
+        id="unit-circle"
+        src={UnitCircleBase}
+        alt="unit circle!!"
+      />
+
+      <img
+        className="unit-circle-markers"
+        id="unit-markers"
+        src={UnitCircleMarkers}
+        alt="unit markers"
+      />
+
+      <img
+        className="unit-circle-turtle"
+        id="unit-turtle"
+        src={UnitCircleTurle}
+        alt="unit turtle"
+        style={{ transform: `rotate(${turtleAngle}deg)` }}
+      />
+
+      <p
+        className={labelOne.type}
+        style={{ position: 'absolute', left: 920, top: 220 }}
+      >
+        {labelOne.label}
+      </p>
```

Comment:
I'd put style into the css, and use css `id` to specify! And I'd also do a mapping on the object array `labels` or whatever name :))